themes/default/assets/sass/_buttons.scss

```diff
@@ -91,6 +66,10 @@
     &:hover {
         @apply border-gray-700 text-gray-700 bg-transparent;
     }
+
+    i {
+        @apply mr-1;
+    }
```

Comment:
Just adds a bit of breathing room for the buttons [on this page](https://www.pulumi.com/brand/).